data/model/test/test_repo_mirroring.py

```diff
@@ -245,3 +248,10 @@ def test_release_mirror(initialized_db):
     mirror = release_mirror(mirror, RepoMirrorStatus.FAIL)
     assert mirror.sync_retries_remaining == 3
     assert mirror.sync_start_date > original_sync_start_date
+
+
+@pytest.mark.parametrize(
+    "rule_value, cleaned_rule_value", [(["", " 1.1 ", "", "1.1"], ["*", "1.1"])]
```

Comment:
Can you also add the following test case?
```
"1.1, latest, "
```
(Note the trailing comma and extra space)

It *shouldn't* happen with JSON but a quick search reveals that *some* browsers will allow it. I am curious if, as an edge-case, we will accidentally inject `*` into a list of other tags.

Reply:
that's not how it works: a list is expected. if that comes in as a string that's what will be saved.